solr-plugin/src/test/java/org/entando/entando/plugins/jpsolr/web/content/AdvContentSearchControllerTest.java

```diff
@@ -1215,6 +1220,10 @@ void testSearchByOption() throws Exception {
                 this.contentManager.deleteContent(content2Id);
             }
         }
+        synchronized (this) {
```

Comment:
same as previous this could lead to a problem, what if the pipeline is slower than usual?